Readme.txt

```diff
@@ -1,77 +1,138 @@
                                 MORSE RUNNER
                               Contest Simulator
                                   freeware
-
-                   Version 1.80 Field Day Contest Prototype
+                                  
+                        Version 1.82 - ARRL DX Contest
```

Comment:
Hmmm... Not sure of the name here. This will be a hard pattern to follow each release, especially when we have multiple contests being added.
- Initially I was thinking this could be a theme-oriented name, perhaps indicating the goal of the current milestone/release. In this case for v1.82, it is the User Group Announcement Release. So perhaps this line should be `Version 1.82 - User Group Announcement`.
- On the other hand, perhaps we should simply this to be similar to MR splash screen. Perhaps:
```
                                             MORSE RUNNER
                                          Contest Simulator
                                               freeware

                                             Version 1.82
```

Reply:
I like the name here.. My reasoning is that this is for the individual users. They are going to refer to this name instead of the version number when talking to others. The majority of the individual users will be unaware of what the User Group Announcement is pertaining to. When we have multiple contests or features we can simplify the name to be more generic like the Spring 23 version or the Five More Contests version. This also "sells" why they need to upgrade more than a number. When I told KD0EE we had a ARRL DX version coming out he was very excited, not as excited if I said 1.82. It's a cheerleading thing.. lol 

But honestly, I'm easy as Sunday morning on all these changes so let me know what you decide.

Reply:
You convinced me. I see your point and agree. Having this name does make sense to an end user community. Perhaps the name I'm looking at is more of a milestone or goal name. I'm good with your change.